templates/definition/meter/goodwe-hybrid.yaml

```diff
@@ -92,5 +92,56 @@ render: |
       type: holding
       decode: uint32
     scale: 0.1
+  batterymode:
+    source: watchdog
+    timeout: 30s
+    reset: 1 # reset watchdog on normal
+    set:
+      source: switch
+      switch:
+      - case: 1 # normal
+        set:
+          source: const
+          value: 1 # Normal operation mode
+          set:
+            source: modbus
+            {{- include "modbus" . | indent 10 }}
+            register:
+              address: 47511 # EMSPowerMode
+              type: writesingle
+              encoding: uint16
+      - case: 2 # hold
+        set:
+          source: const
+          value: 8 # Battery standby
+          set: 
+            source: modbus
+            {{- include "modbus" . | indent 10 }}
+            register:
+              address: 47511 # EMSPowerMode
+              type: writesingle
+              encoding: uint16
+      - case: 3 # charge
+        set:
+          source: sequence
+          set:
+          - source: const
+            value: 2 # Charge from PV+AC according to the EMSPowerSet below
+            set: 
+              source: modbus
+              {{- include "modbus" . | indent 12 }}
+              register:
+                address: 47511 # EMSPowerMode
+                type: writesingle
+                encoding: uint16
+          - source: const
+            value: 10000 # For charge battery mode: Maximum allowed power from Grid in W 
```

Comment:
Should make this magic value configurable as `maxForcedChargePower`

Reply:
Ok, i see a benefit there. Should i try and do the change for this device only? 
Or should we make a general PR where we specifically adapt the whole project for this new configurable value for all devices supporting battery charging?

Reply:
Soll ich das als Arbeitsauftrag verstehen? 😀

Reply:
Sehr gerne 😎

Reply:
> Should make this magic value configurable as maxForcedChargePower

Das trifft hier so nicht zu und ich sehe keinen Sinn darin dies konfigurierbar zu machen.
Es wird mit maximaler Leistung (entsprechend des BMS) geladen und fertig. Der Wert ist nur ein Alibi für "so viel wie möglich".
Wir können das daher einfach so lassen wie es ist. Ist bei anderen Implementierungen auch nicht anders.
Am Ende würde da nur irgendwer wieder dran rumspielen und sich wundern warum es Fehlermeldungen oder komisches Verhalten gibt.